### PR TITLE
removed unnecessary variants

### PR DESCRIPTION
**Pull request by gerouvi**
Removed unnecessary variants in inputs and text area, and I moved the styles from the auth variant to baseStyles so that all inputs/text areas have the same styles. If we need different styles in the future, we will create a variant.

### CODE REVIEW

**Review by jpaulet — APPROVED** [review]

File: src/components/AccountSaas/Create.tsx [hunk 1]
```diff
@@ -134,13 +133,7 @@ export const AccountCreate = ({ children, ...props }: FlexProps) => {
...
               {...register('description')}
               placeholder={t('form.account_create.description_placeholder').toString()}
-              _focus={{
-                boxShadow: 'none',
-
-                _hover: {
-                  boxShadow: 'none',
-                },
-              }}
+              variant='ss'
```
Comment (selankon) [inline src/components/AccountSaas/Create.tsx:136]: Variant ss??

File: src/components/Auth/SignUp.tsx [hunk 2]
```diff
@@ -78,15 +78,15 @@ const SignUp = () => {
...
+              <Input {...register('firstName', { required })} type='text' placeholder='First Name' />
               <FormErrorMessage>{errors.firstName?.message?.toString()}</FormErrorMessage>
             </FormControl>
             <FormControl isInvalid={!!errors.lastName} flexGrow={1} mb='24px'>
               <FormLabel display='flex' ms='4px' fontSize='sm' fontWeight='500' color={textColor} mb='8px'>
                 {t('signup_last_name')}
                 <Text color={textColorBrand}>*</Text>
               </FormLabel>
-              <Input {...register('lastName', { required })} variant='auth' type='text' placeholder='Last Name' />
+              <Input {...register('lastName', { required })} type='text' placeholder='Last Name' />
```
Comment (selankon) [inline src/components/Auth/SignUp.tsx:89]: placeholders should be translated using `t` function

**Review by selankon — CHANGES_REQUESTED** [review]

File: src/components/Auth/SignUp.tsx [hunk 3]
```diff
@@ -78,15 +78,19 @@ const SignUp = () => {
                 {t('signup_first_name')}
                 <Text color={textColorBrand}>*</Text>
               </FormLabel>
-              <Input {...register('firstName', { required })} variant='auth' type='text' placeholder='First Name' />
+              <Input {...register('firstName', { required })} type='text' placeholder='First Name' />
```
Comment (selankon) [inline src/components/Auth/SignUp.tsx:81]: This placeholder is still without translation :S

**Review by selankon — CHANGES_REQUESTED** [review]

File: src/components/AccountSaas/Create.tsx [hunk 4]
```diff
@@ -134,13 +133,6 @@ export const AccountCreate = ({ children, ...props }: FlexProps) => {
             <Textarea
               {...register('description')}
               placeholder={t('form.account_create.description_placeholder').toString()}
```
Comment (selankon) [inline src/components/AccountSaas/Create.tsx:135]: Unnecessary tostring

**Review by selankon — CHANGES_REQUESTED** [review]

**Review by selankon — APPROVED** [review]

**Pull request merged**